runtimes/common/src/bonded_currencies/hooks.rs

```diff
@@ -0,0 +1,56 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use frame_support::{
+	pallet_prelude::{OptionQuery, PalletInfoAccess},
+	storage_alias,
+};
+use pallet_bonded_coins::{traits::NextAssetIds, FungiblesAssetIdOf};
+use sp_runtime::{ArithmeticError, BoundedVec, DispatchError};
+use sp_std::{marker::PhantomData, vec::Vec};
+
+use crate::bonded_currencies::AssetId;
+
+#[storage_alias]
+pub type NextAssetId<BondingPallet: PalletInfoAccess, T> =
+	StorageValue<BondingPallet, FungiblesAssetIdOf<T>, OptionQuery>;
```

Comment:
I guess using `ValueQuery` and hence returning `0` would probably be a better idea.

Reply:
I can remember, I had a strange error when I was using `ValueQuery`. But It seems to compile today: [d95534e](https://github.com/KILTprotocol/kilt-node/pull/818/commits/d95534ea19ddd0031a10a8c5faef9df932954c50)

Not sure what was wrong.